script.js

```diff
@@ -12,42 +12,33 @@ const country_list = ["Afghanistan","Albania","Algeria","Andorra","Angola","Angu
 	,"Turkey","Turkmenistan","Turks and Caicos","Uganda","Ukraine","United Arab Emirates","United Kingdom","United States","United States Minor Outlying Islands","Uruguay"
   ,"Uzbekistan","Venezuela","Vietnam","Virgin Islands (U.S.)","Yemen","Zambia","Zimbabwe"];
 
-  function getRandomIndex(items) {
+// API
+const api= "https://restcountries.eu/rest/v2/all"

```

Comment:
```suggestion
const api= "https://restcountries.eu/rest/v2/all"

```

Name `api` does not make sense. This is a URL so name it accordingly.

Reply:
fixed it